pelicanconf.py

```diff
@@ -96,6 +96,6 @@ def page_with_slug(pages, slug):
         continue
     entries = os.listdir(path)
     for entry in entries:
-        # version must either be "latest" or "x.x.x"
-        if re.match('^((?i:latest)|\d+\.\d+\.\d+)$', entry):
+        # version must either be "latest" or "x.x"
+        if re.match('^((?i:latest)|\d+\.\d+)$', entry):
```

Comment:
I think if we ever decide we need a separate set of docs for a patch release or mess with how this gets invoked, this'll silently break again
* Would it not make more sense to use `(\d+\.\d+(\.\d+)?)` instead?
* Would it not be better to fail with an error if the version string doesn't match the pattern (that would be in line with the "must either be ...")

Reply:
How about we remove the regex altogether? Nothing should be in that folder except documented versions as they are automatically pushed from other repositories.

Reply:
Sounds good to me!

Reply:
Sure. We can always reinstate the regex if something else will be stored there too right(?)